tests/unit_tests/pyanaconda_tests/modules/storage/test_module_device_tree.py

```diff
@@ -621,6 +621,39 @@ def test_get_file_system_free_space(self):
         assert total_size < Size("10 GiB").get_bytes()
         assert total_size > Size("8 GiB").get_bytes()
 
+    def test_get_free_space_for_system(self):
+        """Test GetFreeSpaceForSystem."""
+        self._add_device(StorageDevice(
+            "dev1",
+            fmt=get_format("ext4", mountpoint="/"),
+            size=Size("5 GiB"))
+        )
+
+        self._add_device(StorageDevice(
+            "dev2",
+            fmt=get_format("ext4", mountpoint="/usr"),
+            size=Size("5 GiB"))
+        )
+
+        # /home should not be counted in
+        self._add_device(StorageDevice(
+            "dev3",
+            fmt=get_format("ext4", mountpoint="/home"),
+            size=Size("5 GiB"))
+        )
+
+        total_size = self.interface.GetFreeSpaceForSystem([])
+        assert total_size == 0
+
+        total_size = self.interface.GetFileSystemFreeSpace(["/", "/usr"])
```

Comment:
Indeed, the bot is probably right, it looks here you wanted to check instead GetFreeSpaceForSystem